release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -651,6 +651,15 @@ You must still use the `performance-addon-operator-must-gather` image when runni
 [id="ocp-4-11-insights-operator"]
 === Insights Operator
 
+==== Insights Operator data collection enhancements
+In OpenShift Container Platform 4.11, the Insights Operator collects the following additional information:
+
+* Cluster `images.config.openshift.io` resource definitions
+* Error messages from the `kube-controller-manager` container logs 
```

Comment:
I would say `Specific error messages .....` Since we don't gather everything. See https://github.com/openshift/insights-operator/blob/master/pkg/gatherers/clusterconfig/kube_controller_manager_logs.go#L35

Reply:
Hmm yeah I didn't say "Specific error messages" because that is kinda vague... Maybe we can say something like: 
`kube-controller-manager container logs when the "Internal error occurred: error resolving resource" or "syncing garbage collector with updated resources from discovery" error messages are present` ?

Reply:
Hmm OK....I thought some level of vagueness is OK :) Another option is to link the https://github.com/openshift/insights-operator/blob/master/docs/gathered-data.md#kubecontrollermanagerlogs

Reply:
We can't link github without PX approval and I'm not sure this rises to the right level for it here : /

Reply:
OK :)